app/plugins/system_controller/system/index.js

```diff
@@ -4,6 +4,7 @@ var libQ = require('kew');
 var fs = require('fs-extra');
 var config = new (require('v-conf'))();
 var execSync = require('child_process').execSync;
+const { spwan } = require('child_process');
```

Comment:
While this is technically correct, it's inconsistent with current coding style. Please rephrase it as 
var spawn = require('child_process');

Reply:
Oops, sorry for the syntax, was copy pasting over from the plugin.. Will fix!